src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingObjectMatchProcessor.cs

```diff
@@ -0,0 +1,111 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.SqlTools.ServiceLayer.Scripting.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.Scripting
+{
+    public static class ScriptingObjectMatchProcessor
```

Comment:
Fixed

---
In reply to: [111650616](https://github.com/Microsoft/sqltoolsservice/pull/316#discussion_r111650616) [](ancestors = 111650616)